src/templates.ts

```diff
@@ -7,22 +7,17 @@ export interface Templates<Req extends Request = Request, Res extends Response =
      *
      * All of them should be rendered as-is, without any escaping for the renderer to work.
      */
-    renderMain(req: Req, res: Res, params: MainTemplateParams): string;
+    renderMain: (req: Req, res: Res, params: MainTemplateParams) => string;
```

Comment:
Is this not all deprecated then ?

Reply:
Oh yea we could deprecate it instead if you prefer? Or should we go direct to breaking release?